.github/workflows/sanity.yml

```diff
@@ -26,9 +26,13 @@ jobs:
     - name: Lint
       run: make lint
     - name: Fmt
-      run: make fmt
+      run: |
+        make fmt && git diff --exit-code
     - name: Vet
       run: make vet
+    - name: Verify go modules are properly up-to-date
+      run: |
+        make mod-tidy mod-verify && git diff --exit-code
```

Comment:
One question: Is this guaranteed to run `mod-tidy` and then `mod-verify`?

Reply:
btw, would it make sense to merge `mod-tidy` and `mod-verify` into one target? They seem to always used together.

Reply:
`mod-tidy` can also be used during development before checking things in, so it may make sense to keep them separate.

Reply:
> mod-tidy can also be used during development before checking things in

Considering that there is a job that runs both targets, what's the point to run only one before checking things in?

Reply:
> One question: Is this guaranteed to run `mod-tidy` and then `mod-verify`?

You mean if both are run, is it guaranteed that they are run in that order ? Yes, unless there is a direct or indirect dependency between the two targets, in which case they would be always run in dependency order.

Reply:
And for us, those are both phony targets without any prereqs/dependencies.

Reply:
Actually, I have to say that now that I looked at `go mod help verify`, I'm not sure if it makes any sense to run `go mod verify` in a workflow job if/where everything start afresh from scratch...

Reply:
Yes, I think `make mod-tidy` is sufficient in this case.

Reply:
Okay, I'll update the PR...

Reply:
Updated.